test/skimNtuple2018_HHbtag.cpp

```diff
@@ -1254,11 +1254,11 @@ int main (int argc, char** argv)
     int idx2hs_b = -1;     // bjet-2 index
     TLorentzVector vGenB1; // bjet-1 tlv
     TLorentzVector vGenB2; // bjet-2 tlv
-
+    TLorentzVector vH1, vH2;
```

Comment:
I'm not sure I understand why `vH1` and `vH2` have to be defined outside the `if (isHHsignal || HHrewType != kNone)`

Reply:
True, they don't have do be defined outside. I only did that because the other gen branches for the b quarks etc are filled at a later stage, but the Higgs ones could already be filled in this if statement. Shall I change that?

Reply:
that was mostly out of curiosity, but yes, if it's OK for you, you can indeed put back the previous initialization (and maybe make it consistent in the 2017 skimmer).

Reply:
I moved vH1 and vH2 back to the if statement and also fill the branches depending on them directly there

Reply:
thank you!